applications/luci-app-email/htdocs/luci-static/resources/view/email/emailrelay.js

```diff
@@ -0,0 +1,218 @@
+			_('Mode: --as-server or --as-proxy. The cmdline means append extra_cmdline'));
+		o.value('server', _('Server: receive email'));
+		o.value('proxy', _('Proxy: receive email, store and forward'));
+		o.value('cmdline', _('Command line (deprecated)'));
+		o.default = 'server';
+		o.rmempty = false;
+
+		o = s.taboption('smtp_server', form.Value, 'port', _('Port'),
+			_('SMTP Port to listen incoming emails'));
+		o.datatype = 'port';
+		o.default = 25;
+
+		o = s.taboption('smtp_server', form.Flag, 'remote_clients', _('Allow remote clients'),
+			_('By default only local connections allowed. Check your firewall to avoid spam. See --remote-clients'));
+		o.modalonly = true;
+		o.default = '0';
+
+		o = s.taboption('smtp_server', form.Flag, 'anonymous', _('Anonymous'),
+			_('Disables the server\'s SMTP VRFY command'));
+		o.modalonly = true;
+		o.default = '0';
+
+		o = s.taboption('smtp_server', form.Value, 'domain', _('Domain'),
+			_('Specifies the network name that is used in SMTP EHLO. The default is derived from a DNS lookup of the local hostname'));
+		o.datatype = 'hostname';
+		o.optional = true;
+		o.rmempty = false;
+
+		o = s.taboption('smtp_server', form.Value, 'server_auth', _('Auth file'),
+			_('Server/proxy authorization file. See --server-auth'));
+		o.datatype = 'file';
+		o.default = '/etc/emailrelay.auth';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+
+		o = s.taboption('smtp_server', form.Value, 'dnsbl', _('DNSBL'),
+			_('Comma separated config of DNS Block List. Starts with the transport DNS server:port, a timeout, threshold and servers.\n' +
+				'See --dnsbl and Connection blocking in reference'));
+		o.datatype = 'string';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+		o.placeholder = '127.0.0.1:53,500,1,dbl.spamhaus.org,bl.0spam.org';
+
+		o = s.taboption('smtp_server', form.Flag, 'server_tls', _('Use TLS for server/proxy'),
+			_('Use TLS encryption for SMTP'));
+		o.default = '0';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+
+		o = s.taboption('smtp_server', form.Value, 'server_tls_key', _('TLS private key'),
+			_('Path to TLS private key. E.g. /etc/ssl/acme/example.com.key'));
+		o.datatype = 'file';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+		o.depends('server_tls', '1');
+
+		o = s.taboption('smtp_server', form.Value, 'server_tls_certificate', _('TLS certificate'),
+			_('Path to TLS cert. E.g. /etc/ssl/acme/example.com.fullchain.crt'));
+		o.datatype = 'file';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('server_tls', '1');
+
+		o = s.taboption('smtp_server', form.Value, 'server_tls_verify', _('CA certificate'),
+			_('Enables verification of remote SMTP and POP clients\' certificates against any of the trusted ' +
+				'CA certificates in the specified file or directory. In many use cases this should be a file containing ' +
+				'just your self-signed root certificate. ' +
```

Comment:
Please split the translation in two after '.'
And please fix the alignment oft translation.